toolkits/springer/packages/springer-media/demo/main.scss

```diff
@@ -2,8 +2,6 @@
 
 // we must not add `node_modules/` to the Sass includePaths config,
 //   so use relative paths here. Paths should be relative to this file.
-@import '../node_modules/@springernature/brand-context/default/scss/core.scss';
-@import '../node_modules/@springernature/brand-context/default/scss/enhanced.scss';
```

Comment:
each of the imprint brands (nature, springer, springernature) imports default the builds on it, so no need to import it here